src/dashboard/src/main/management/commands/rebuild_aip_index_from_storage_service.py

```diff
@@ -131,32 +131,36 @@ def handle(self, *args, **options):
         if options["uuid"]:
             package_uuid = options["uuid"]
             self.info("Reindexing package {}".format(package_uuid))
-            package_info = storageService.get_file_info(uuid=package_uuid)
+            try:
+                package_info = storageService.get_file_info(uuid=package_uuid)[0]
+            except IndexError:
+                self.error(
+                    "Error indexing package {0}. "
+                    "Package with supplied UUID not found in Storage Service".format(
+                        package_uuid
+                    )
+                )
+                return
 
             is_aic = False
             package_type = package_info.get("package_type")
             if package_type and package_type == "AIC":
                 is_aic = True
 
-            (index_success, error_message) = self.process_package(
+            index_success = self.process_package(
                 es_client,
                 package_info,
                 temp_dir,
                 delete_before_reindexing,
                 is_aic=is_aic,
             )
-            if index_success is True:
-                self.info("Successfully reindexed package {}".format(package_uuid))
-            else:
-                self.error(
-                    "Error indexing package {0}. Details: {1}".format(
-                        package_uuid, error_message
-                    )
-                )
 
             self.info("Cleaning up")
             shutil.rmtree(temp_dir)
-            self.info("Indexing complete.")
+            if index_success:
+                self.success("Success! Indexing complete.")
+            else:
+                self.error("Failure. Indexing complete.")
```

Comment:
Shall we continue to test for `True` here like in the original line at `148`? If it isn't true, can we return immediately after 147? Or do we still need to commit to clean up? Is there any more information that the user (or us) need here, will we get enough information to track back from: `Failure. Indexing complete.` do you think?

Reply:
I think this is cleaned up a bit now. The messages here were a bit vague because `process_package` outputs more detailed information for each package to the terminal as well.